static/app/views/dashboardsV2/widgetBuilder/utils.tsx

```diff
@@ -82,6 +82,30 @@ export function mapErrors(
   return update;
 }
 
+export function flattenErrors(
```

Comment:
since this function is not specific to the `widgetBuilder`, should we move it to a more common file? perhaps https://github.com/getsentry/sentry/blob/master/static/app/views/dashboardsV2/utils.tsx ?

Reply:
is this something that could be useful elsewhere in the app? maybe we could make it a common utility function too

Reply:
Yeah, I can move it to dashboards utils!